cmd/lakectl/cmd/local_commit.go

```diff
@@ -120,7 +124,7 @@ var localCommitCmd = &cobra.Command{
 		}()
 		sigCtx := localHandleSyncInterrupt(cmd.Context(), idx, string(commitOperation))
 		s := local.NewSyncManager(sigCtx, client, syncFlags.parallelism, syncFlags.presign)
-		err = s.Sync(idx.LocalPath(), remote, c)
+		err = s.Sync(idx.LocalPath(), remote, c, contentType, includeContentType)
```

Comment:
I don't understand.  When would I want to set contentType to the _same_ value on all the files in a Sync?